java/arcs/core/host/ArcHostContextParticle.kt

```diff
@@ -46,7 +46,7 @@ class ArcHostContextParticle : AbstractArcHostParticle() {
                         storageKey = handle.value.storageKey.toString(),
                         mode = handle.value.mode.name,
                         type = handle.value.type.tag.name,
-                        ttl = handle.value.ttl?.minutes?.toDouble() ?: 0.0
+                        ttl = handle.value.ttl?.minutes?.toDouble() ?: -1.0
```

Comment:
rather than have a magic number here, i think we should define a constant on the Ttl class for this (if there isn't one already)

Reply:
there isn't, but I'll add one.